<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VotingEntry</class>
 <widget class="QFrame" name="VotingEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <item row="3" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Preference Vote &amp;For:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>payTo</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <layout class="QHBoxLayout" name="payToLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QValidatedLineEdit" name="payTo">
       <property name="toolTip">
        <string>The address to send the payment to (e.g. 1NS17iag9jJgTHD1VXjvLCEnZuQ3rJDE9L)</string>
       </property>
       <property name="maxLength">
        <number>34</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addressBookButton">
       <property name="toolTip">
        <string>Choose address from address book</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pasteButton">
       <property name="toolTip">
        <string>Paste address from clipboard</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteButton">
       <property name="toolTip">
        <string>Remove this recipient</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="payAmount">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="maxVisibleItems">
      <number>9</number>
     </property>
     <item>
      <property name="text">
       <string notr="true">1st</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2nd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3rd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4th</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5th</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6th</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7th</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8th</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9th</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
